src/components/Overview/Overview.react.js

```diff
@@ -149,8 +148,7 @@ class Overview extends Component {
       }
     const actions = <RaisedButton
       label="Cancel"
-      backgroundColor={
-        UserPreferencesStore.getTheme()==='light' ? '#4285f4' : '#19314B'}
+      backgroundColor='#607D8B'
```

Comment:
No, #607D8B is working fine. I think earlier code was not right. I have testes it, its working